File: src/#1455/solution.go
package solution

import "strings"

func isPrefixOfWord(sentence string, searchWord string) int {
	words := strings.Split(sentence, " ")
	ans := -1
	swLen := len(searchWord)
	for i, s := range words {
		if len(s) >= swLen {
			if s[:swLen] == searchWord {
				ans = i + 1
				break
			}
		}
	}
	return ans
}

File: src/#1455/solution.js
var isPrefixOfWord = function (sentence, searchWord) {
  const words = sentence.split(' ')
  const swLen = searchWord.length
  let ans = -1
  console.log(words)
  for (let i = 0; i < words.length; i++) {
    if (words[i].length >= swLen) {
      if (words[i].slice(0, swLen) === searchWord) {
        ans = i + 1
        break
      }
    }
  }
  return ans
}
